data/scenarios/02Tutorials/10-scan.yaml

```diff
@@ -0,0 +1,68 @@
+name: Scan
+description: |
+  Learn how to send robots to safely scan your surroundings.
+entities:
+  - name: Goal
+    display:
+      attr: device
+      char: 'X'
+    description:
+    - When you land on an alien planet, all the entites in the
+      world will be unfamiliar to you.
+    - But you can learn what they are using the scanner device.
+    - Send robots to move to the unknown entites (marked as '?'),
+     'scan forward' and after that come back and 'upload base'.
+    - |
+      TIP: You can also 'scan left' or 'scan north', to save on a few
+      'turn' commands.  
+win: |
+  try {
+    bm <- as base {knows "mountain"};
+    bt <- as base {knows "tree"};
+    bw <- as base {knows "wavy water"};
+    return (bm || bt || bw)
```

Comment:
Should this be `&&` instead of `||`?  Or was this intentional?

Reply:
It was, I was thinking that in the distant future there could be an alternative harder goal where you have to update all at once. Some way to specify achievements, like the stars in the Angry Birds games.

But most importantly it is a lot easier now. :smile: